src/contextwatchdog.js

```diff
@@ -0,0 +1,540 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module watchdog/contextwatchdog
+ */
+
+/* globals console */
+
+import Watchdog from './watchdog';
+import EditorWatchdog from './editorwatchdog';
+import areConnectedThroughProperties from './utils/areconnectedthroughproperties';
+import getSubNodes from './utils/getsubnodes';
+
+/**
+ * The Context Watchdog class.
```

Comment:
I am not sure if you understood my comment. I was suggesting changing the description to one of those that I suggested. I guess I will make my choice then.

Reply:
My bad, I've overlooked it.